salt/metalk8s/addons/ui/deployed/ui.sls

```diff
@@ -59,4 +59,4 @@ Create ui-branding ConfigMap:
           namespace: metalk8s-ui
         data:
           theme.json: |
-           {"theme": {"light":{"brand":{"base": "#607080", "primary": "#FAF9FB", "primaryDark1": "#F7F6F9", "primaryDark2": "#EDEAF0", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#ffffff", "backgroundBluer": "#ECF4FF", "textPrimary": "#313B44", "textSecondary": "#8593A0", "textTertiary": "#6C8398", "borderLight": "#A5A5A5", "border": "#A5A5A5"},"logo_path": "/brand/assets/branding-light.svg"},"dark": {"base": "#6A7B92", "primary": "#1D1D1F", "primaryDark1": "#171718", "primaryDark2": "#0A0A0A", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#121214", "backgroundBluer": "#182A41", "textPrimary": "#FFFFFF", "textSecondary": "#A8B5C1", "textTertiary": "#A7B6C3", "borderLight": "#2C3137", "border": "#A5A5A5"},"logo_path":"/brand/assets/branding-dark.svg"},"custom":{}},"default": "light"}
+           {"theme": {"light":{"brand":{"base": "#607080", "primary": "#FAF9FB", "primaryDark1": "#F7F6F9", "primaryDark2": "#EDEAF0", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#ffffff", "backgroundBluer": "#ECF4FF", "textPrimary": "#313B44", "textSecondary": "#8593A0", "textTertiary": "#6C8398", "borderLight": "#A5A5A5", "border": "#A5A5A5"},"logo_path": "/brand/assets/branding-light.svg"},"dark": {"base": "#6A7B92", "primary": "#1D1D1F", "primaryDark1": "#171718", "primaryDark2": "#0A0A0A", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#121214", "backgroundBluer": "#182A41", "textPrimary": "#FFFFFF", "textSecondary": "#A8B5C1", "textTertiary": "#A7B6C3", "borderLight": "#2C3137", "border": "#A5A5A5"},"logo_path":"/brand/assets/branding-dark.svg"},"custom":{},"default": "light"}
```

Comment:
We **really** need to stop writing this file manually. IIRC, already commented about this some time ago.

Reply:
Yes. agree!!!! I will create a dept ticket.
We will do something in the buildchain to copy the theme from `ui/public/...` to configMap.